bin/datatool/src/args.rs

```diff
@@ -172,6 +185,18 @@ pub(crate) struct SubcParams {
         description = "permit blank proofs after timeout in millis (default strict)"
     )]
     pub(crate) proof_timeout: Option<u32>,
+
+    #[argh(
+        option,
+        description = "fee in sats for reveal transactions. Will be determined dynamically if None"
+    )]
+    pub(crate) fixed_fee: Option<u64>,
```

Comment:
Also, could you also rename this attribute to something like `reveal_fee`? Right now it's not clear what fee it is referring to.